config/LocalSettings_local.php

```diff
@@ -0,0 +1,8 @@
+<?php
+// If not running MediaWiki, exit
+if ( !defined( 'MEDIAWIKI' ) ) {
+	exit;
+}
+#$wgServer = "http://kkk";
```

Comment:
Please refrain from using `KKK` in any context, **ever**. If you are unsure why I am saying this, please look up what the KKK is on Wikipedia.